adls-plugins/src/main/java/co/cask/hydrator/plugin/sink/ADLSBatchSink.java

```diff
@@ -180,8 +180,8 @@ protected void validate(Schema inputSchema) {
           String fieldName = outputField.getName();
           Schema.Field inputField = inputSchema.getField(outputField.getName());
           if (inputField == null) {
-            throw new IllegalArgumentException("Input record does not contain the [" + fieldName
-                                                 + "] field in output schema.");
+            throw new IllegalArgumentException("Input schema does not contain the " + fieldName
```

Comment:
as originally commented, `Input schema does not contain the " + fieldName " field.`